src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -228,7 +248,7 @@ public function getContactsAction($id, Request $request)
      */
     public function getAddressesAction($id, Request $request)
     {
-        if ('true' == $request->get('flat')) {
+        if ('true' === $request->get('flat')) {
```

Comment:
i am a bit afraid of changing this because of bc

```suggestion
        if ('true' == $request->get('flat')) {
```